### PR TITLE
Insights in a standalone app

### PR DESCRIPTION
**Pull request by MartinaeyNL**
New functionality for deploying the insights page in a standalone app.

Changelog standalone app:
- Updated and migrated app to newest version of `or-dashboard-builder` component.
- Added leftside drawer with list of dashboards. Also includes animations and mobile friendly controls.
- App now supports specifying dashboard ID in URL, same as on the Managers' Insights page.
- Realm styling such as buttons and images are now included and being listened for.

.
Changelog general:
- Added loading indicator to initial dashboard load, on both standalone app and manager)
- `or-gauge` component is now more responsive, especially in small sizes.
- Hotfix for chart background being black on mobile.
- Hotfix for Error 500 on chart/datapoints where a long rendering time caused an invalid chart pixel width to be used.
- Attempt to fix datapoints being pulled twice or three times when pressing "reload" button or when any other full rerender occurs.

### CODE REVIEW

**Comment by MartinaeyNL** [comment]
Added changelog to initial comment

**Comment by MartinaeyNL** [comment]
Did some extra fixes, did lots of manual testing, and this branch is now ready for review and merge.

File: ui/app/insights/info.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "consoleAppIncompatible": false,
+  "realms": ["master", "smartcity"],
```
Comment (richturner) [inline ui/app/insights/info.json:3]: This shouldn't be here; app is compatible with any realm

**Comment by MartinaeyNL** [comment]
Just verified the deletion of `info.json` on a deployment and it works fine.

**Review by richturner — APPROVED** [review]

**Pull request merged**